synapse/models/biz.py

```diff
@@ -30,26 +30,26 @@ def getModelDefs(self):
                     'doc': 'A service which is performed by a specific organization.',
                 }),
                 ('biz:service:type:taxonomy', ('taxonomy', {}), {
-                    'doc': 'A taxonomy of service offering types.',
                     'interfaces': ('meta:taxonomy',),
+                    'doc': 'A hierarchical taxonomy of service types.',
                 }),
-                ('biz:dealstatus', ('taxonomy', {}), {
-                    'doc': 'A deal/rfp status taxonomy.',
+                ('biz:deal:status:taxonomy', ('taxonomy', {}), {
```

Comment:
I would vote for consistency here re:type:taxonomy.

I see "status" actually working well as a taxonomy (but only using the top level) since it seems like it would give people more flexibility than the enums in that they can create their own, but also be less squishy than strings. However, this all goes out the window in the event someone tries to create a hierarchy, which I think might get wild.

Reply:
The status of a deal is not the same as the type of deal. Not all taxonomies will be used for `:type` properties, it's just one of the more common use cases. Another "non type" is `risk:alert:verdict`.